src/components/panel/panel.spec.js

```diff
@@ -643,6 +606,226 @@ describe('$mdPanel', function() {
       expect(scrollMaskEl).not.toExist();
       expect($mdUtil.disableScrollAround).toHaveBeenCalled();
     });
+
+    describe('animation hooks: ', function() {
+      it('should call onDomAdded if provided when adding the panel to the DOM',
+          function() {
+            var onDomAdded = jasmine.createSpy();
+            var config = angular.extend(
+                {'onDomAdded': onDomAdded}, DEFAULT_CONFIG);
+
+            panelRef = $mdPanel.create(config);
+            panelRef.attach();
+
+            flushPanel();
+
+            expect(onDomAdded).toHaveBeenCalled();
+            expect(PANEL_EL).toExist();
+            expect(PANEL_WRAPPER_CLASS).toHaveClass(HIDDEN_CLASS);
+          });
+
+      it('when onDomAdded resolves continue resolving the ' +
```

Comment:
This needs to be of the form: "should continue resolving when onDomAdded resolves"